### PR TITLE
Added careplans, goals history and diagnoses history explores

### PR DESCRIPTION
**Pull request by jagrutz**
https://alayacare.atlassian.net/browse/SPX-1834

### CODE REVIEW

File: looker_config.json [hunk 1]
```diff
@@ -113,6 +113,24 @@
...
+    },
+    "careplans": {
+      "prod_ready": false,
+      "label": "Careplans",
+      "view": "careplans",
+      "release_flag": "SPX-1834-careplans_feature"
+    },
+    "care_plan_goals_history": {
+      "prod_ready": false,
+      "label": "Careplans",
```
Comment (Mathieu-Gilli) [inline looker_config.json:125]: Please change label

**Review by Mathieu-Gilli — CHANGES_REQUESTED** [review]
Please change label

File: looker_config.json (hunk 1 above)
Comment (jagrutz) [reply looker_config.json]: Labels updated

**Review by estheban — APPROVED** [review]
LGTM

**Review by Mathieu-Gilli — APPROVED** [review]

**Pull request merged**